src/components/dashboard/components/dictionary/DictionaryCard.jsx

```diff
@@ -45,10 +49,18 @@ const DictionaryCard = (dictionary)=> {
           </div>
         </div>
         <div className="source-card-footer">
+          <button
+            type="button"
+            className="btn btn-outline-secondary"
+            id="viewDetails"
+            onClick={() => fetchData({ url })}
+          >
+            View Details
+          </button>
         </div>
       </div>
     </div>
-  );
-};
+  )
```

Comment:
semicolons in line 63, 64 and 66 are missing

Reply:
Done